targets/particle/README.md

```diff
@@ -0,0 +1,91 @@
+### About
+
+This folder contains files to run JerryScript beside Particle Device Firmware on Photon board.
+It  runs a mini example, blinking an LED which is the "Hello World" example of the microcontroller universe.
+
+### How to build
+
+#### 1. Preface / Directory structure
+
+Assume `harmony` as the path to the projects to build.
+The folder tree related would look like this.
+
+```
+harmony
+  + jerryscript
+  |  + targets
+  |      + particle
+  + particle
+  |  + firmware
+```
+
+
+#### 2, Update/Install the Particle Firmware
+
+For detailed descriptions please visit the official website of the firmware: https://www.particle.io/
+
+You can checkout the firmware with the following command:
+
+```
+# assume you are in harmony folder
+git clone https://github.com/spark/firmware.git particle/firmware particle/firmware
+````
+
+The Photon’s latest firmware release is hosted in the latest branch of the firmware repo.
+
+```
+# assume you are in harmony folder
+cd particle/firmware
+git checkout latest
+```
+
+Before you type any commands, put your Photon in DFU mode: hold down both the SETUP and RESET buttons. Then release RESET, but hold SETUP until you see the RGB blink yellow. That means the Photon is in DFU mode. To verify that the Photon is in DFU mode and dfu-util is installed properly, try the dfu-util -l command. You should see the device, and the important parts there are the hex values inside the braces – the USB VID and PID, which should be 2b04 and d006 for the Photon.
+
+To build and flash the firmware: switch to the modules directory then call make with a few parameters to build and upload the firmware:
+
+```
+cd modules
+make PLATFORM=photon clean all program-dfu
+```
+
+#### 3. Build JerryScript
+
+```
+# assume you are in harmony folder
+cd jerryscript
+make -f ./targets/particle/Makefile.particle
+```
+
+This will create a binary file in the `/build/particle/` folder:
+```
+jerry_main.bin
+```
+
+That’s the binary what we’ll be flashing with dfu-util.
+
+
+#### 4. Flashing
+
+Make sure you put your Photon in DFU mode.
```

Comment:
It would be good to add a short description how to put the device into DFU mode.